validator/entrypoint.sh

```diff
@@ -16,6 +16,18 @@ if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_P
   fi
 fi
 
+
+
+if [[ "$EXIT_VALIDATOR" == "I want to exit my validators" ]];
```

Comment:
This env should be defined in the `compose` file so user can set  value

Reply:
https://github.com/dappnode/DAppNodePackage-teku-prater/pull/88/commits/f879b3cc5c1bc307350f1ff2313fad6f977cb445 its defined there, initially its empty